templates/cluster-template-aws.yaml

```diff
@@ -20,6 +20,7 @@ metadata:
   name: ${CLUSTER_NAME}
 spec:
   region: ${AWS_REGION}
+  imageLookupBaseOS: ubuntu-20.04
```

Comment:
Great find!
Does kubernetes version > 1.28.1 only work with 20.04? Is there something like ubuntu-latest? We wouldn't want our example to break again when k8s version and AMI image diverge.

Reply:
Great observation. That would be ideal, but I don't think images are published with ubuntu-latest in their name.
See the docs here: https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/v1.0.0/pkg/cloud/services/ec2/ami.go#L38-L53